spec/models/post_spec.rb

```diff
@@ -1,43 +1,61 @@
 
   describe 'validations' do
     it 'should have a title' do
-      post = user.posts.build(title: nil, text: 'One line at a time')
-      expect(post).to_not be_valid
+      subject.title = nil
+      expect(subject).to_not be_valid
     end
 
     it 'should have text' do
-      post = user.posts.build(title: 'Coding in progress', text: nil)
-      expect(post).to_not be_valid
+      subject.text = nil
+      expect(subject).to_not be_valid
     end
 
     it 'title should not be greater than 250' do
-      post = user.posts.build(title: 'Hello world' * 25, text: 'One line at a time')
-      expect(post).to_not be_valid
+      subject.title = 'Hello world' * 25
+      expect(subject).to_not be_valid
     end
 
-    it 'comments_counter should be an integer' do
-      post = user.posts.build(title: 'Coding in progress', text: 'One line at a time', comments_counter: '')
-      expect(post).to_not be_valid
+    it 'comments_counter should be an interger' do
+      subject.comments_counter = ''
+      expect(subject).to_not be_valid
     end
 
-    it 'likes_counter should be an integer' do
-      post = user.posts.build(title: 'Coding in progress', text: 'One line at a time', likes_counter: '')
-      expect(post).to_not be_valid
+    it 'likes_counter should be an interger' do
+      subject.likes_counter = ''
+      expect(subject).to_not be_valid
     end
 
     it 'comments_counter should be greater than or equal to 0' do
-      post = user.posts.build(title: 'Coding in progress', text: 'One line at a time', comments_counter: -1)
-      expect(post).to_not be_valid
+      subject.comments_counter = -1
+      expect(subject).to_not be_valid
     end
 
     it 'likes_counter should be greater than or equal to 0' do
-      post = user.posts.build(title: 'Coding in progress', text: 'One line at a time', likes_counter: -1)
-      expect(post).to_not be_valid
+      subject.likes_counter = -1
+      expect(subject).to_not be_valid
+    end
+  end
+
+  describe '#recent_comments' do
+    commenter2 = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                             bio: 'Teacher from Poland.')
+
+    it 'return most recents 5 comments' do
+      subject.comments.create(author: commenter2, text: 'Hi Tom!')
+      comment2 = subject.comments.create(author: user, text: 'Hi Lilly!')
+      comment3 = subject.comments.create(author: commenter2, text: 'Hope you all done with your project?')
+      comment4 = subject.comments.create(author: user, text: 'I asked after you from Ebun yesterday')
+      comment5 = subject.comments.create(author: commenter2, text: 'Oh, really, thanks.')
+      comment6 = subject.comments.create(author: user, text: 'I am almost done with the project')
+      recent_comments = subject.recent_comments
+
+      expect(recent_comments).to contain_exactly(comment6, comment5, comment4, comment3, comment2)
+      expect(recent_comments.length).to eq(5)
     end
```

Comment:
- [ ] As mentioned by the previous reviewer you should test two model methods in the post model (`update_posts_counter and recent_comments) but you only test the `recent_comment` method. Kindly test the `update_posts_counter` as well 🙏🏾